src/app/home/pages/PlansPage/components/MigrateModal.tsx

```diff
@@ -35,28 +36,32 @@ const MigrateModal: React.FunctionComponent<IProps> = ({ onHandleClose, isOpen,
           </GridItem>
           <GridItem className={styles.gridMargin}>
             <Checkbox
-              label="Don't halt transactions on the source during migration."
+              label="Halt transactions on the source during migration."
               aria-label="halt-label"
               id="transaction-halt-checkbox"
-              isChecked={disableQuiesce}
+              isChecked={enableQuiesce}
               onChange={handleChange}
             />
           </GridItem>
           <GridItem>
             <Grid hasGutter>
```

Comment:
Btw, if your grid has only one item, you don't need to use a Grid. If you're just looking for the gutter, you could instead put some padding on the GridItem parent (`<GridItem className={spacing.pMd}>`)

Reply:
Good to know!